src/index.js

```diff
@@ -43,12 +64,49 @@ async function publishToSQS(data) {
   }
 }
 
+/**
+ * send IndexerNotified event to the appropriate SNS topic
+ * @see https://github.com/elastic-ipfs/metrics-collector/blob/main/src/indexer-events/examples/IndexerNotified.json
+ * @param {object} arg.s3 - s3 object info
+ * @param {string} arg.s3.bucket.name - s3 bucket name
+ * @param {string} arg.s3.object.key - s3 key name
+ * @param {number} arg.s3.object.size - object size in bytes
+ * @param {Date} arg.startTime - time of IndexerNotified event
+ */
+function createIndexerNotifiedEvent({ s3, startTime = new Date() }) {
+  const event = {
+    type: 'IndexerNotified',
+    uri: createS3URL(s3.bucket.name, s3.object.key).toString(),
+    byteLength: s3.object.size,
+    startTime: startTime.toISOString()
+  }
+  return event
+}
+
+/**
+ * Broadcast an Elastic IPFS Event on the appropriate pubsub topic
+ * @param {object} arg.event - elastic-ipfs event to publish
+ * @param {string} arg.eventsTopic - topic to publish event on
+ */
+async function emitIpfsEvent({
```

Comment:
This function does not need to be async. Replace await with return and you're done!